SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialSpanIdCache.h

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
```

Comment:
I don't think you need this. But generally any inclusion of something like core minimal far outweighs any forward declaration you do.